app/javascript/app/components/info-icon/info-icon-component.jsx

```diff
@@ -0,0 +1,35 @@
+import React, { PureComponent } from 'react';
```

Comment:
Could we move this component to the buttons folder? There are so many components in the root that is hard to find what you are looking for.